26.10.22/string/js/hw3_myskovets.js

```diff
@@ -0,0 +1,10 @@
+var button = document.querySelector('button');
+
+button.onclick = function() {
+  var name = prompt('First part');
```

Comment:
Don't use ```var``` in the future, pls. It could make you a headache. Try to use ```let``` and ```const```

Reply:
Yes, Master Dima. I'm on the star of the long way to the top. Will be working hard!!!😂😂😂